core/src/main/java/edu/wpi/grip/core/operations/composite/FindBlobsOperation.java

```diff
@@ -33,6 +33,10 @@
       .createNumberSpinnerSocketHint("Min Area", 1);
   private final SocketHint<List<Number>> circularityHint = SocketHints.Inputs
       .createNumberListRangeSocketHint("Circularity", 0.0, 1.0);
+  private final SocketHint<List<Number>> convexityHint = SocketHints.Inputs
+      .createNumberListRangeSocketHint("Convexity", 0.0, 1.0);
+  private final SocketHint<List<Number>> inertiaRatioHint = SocketHints.Inputs
+      .createNumberListRangeSocketHint("Inertia Ratio", 0.0, 1.0);
   private final SocketHint<Boolean> colorHint = SocketHints
       .createBooleanSocketHint("Dark Blobs", false);
```

Comment:
@SamCarlberg At some point, we should come back through here and make all these static.